docs/oasis-network-primer/token_distribution/Chart.tsx

```diff
@@ -13,9 +13,12 @@ const cumulativeSum = (values: number[]) => {
   return values.map(value => acc += value);
 };
 
+/**
+ * Offset from 2020-11-18
+ */
 const formatMonthOffsets = (monthOffsets: number[]) => {
   return monthOffsets.map(monthOffset => {
-    const date = new Date('2020-11-18');
+    const date = new Date(2020, 10 /* zero-based */, 18);
```

Comment:
:information_source: monthIndex bug, not timezone bug

Reply:
`Date('2020-11-18')` was midnight in UTC, so toLocaleDateString in some timezones returned 2020-11-17.
`Date(2020, 10 /* zero-based */, 18)` is midnight in local timezone, so toLocaleDateString should always return 2020-11-18.

I'd prefer a more intuitive `Date('2020-11-18').toUTCString` but it doesn't have formatting options :cry: